sample/SampleServer/Program.cs

```diff
@@ -35,8 +36,10 @@ static async Task MainAsync(string[] args)
                 options
                     .WithInput(Console.OpenStandardInput())
                     .WithOutput(Console.OpenStandardOutput())
-                    .ConfigureLogging(x => x.AddSerilog())
-                    .AddDefaultLoggingProvider()
+                    .ConfigureLogging(x => x
+                        .AddSerilog()
+                        .AddLanguageServer(LogLevel.Critical)
+                        .SetMinimumLevel(LogLevel.Trace))
```

Comment:
I'd like to avoid having this, but truth is that MEL adds a default filter which works on the factory level, so we won't get our logs to LSP logger if they are < Information (which is default),

Reply:
As long as it's configured by the consumer (read: Host process) I think that's fine.